### PR TITLE
Fix team page based on Tak's requests

### PR DESCRIPTION
**Pull request by loicmurumba**

### CODE REVIEW

**Comment by STBTony** [comment]
@loicmurumba notice how other images taken previously are compressed downto less than 80kb. These images you are pushing are over 6MB so that's not good. Compress them before pushing. And for future reference - to not slow down the repo, revert instead of commiting deleted changes since simply commiting deleted images does not erase the history from the repo. This time ill give it a pass. Compress the image and commit and push is fine.

File: js/team.js [hunk 1]
```diff
@@ -44,116 +44,92 @@ function _goBack() {
...
 	{
-		name: 'Iris Chang',
-		program: '2nd Year Systems Design Engineering',
-		title: 'Business Manager',
-		imgSrc: 'Iris_Chang'
-	},
-	{
 		name: 'Toni Li',
-		program: '2nd Year Management Engineering',
+		program: '3rd Year Management Engineering',
```
Comment (STBTony) [inline js/team.js]: Check with Toni cuz i don't think this information is correct

Comment (STBTony) [reply js/team.js]: @loicmurumba

Comment (loicmurumba) [reply js/team.js]: @STBTony I talked to Tak and he thinks we should remove the years, I agree too

**Review by STBTony — APPROVED** [review]
LGTM

**Pull request merged**